README.md

```diff
@@ -15,7 +15,7 @@ engines:
     config:
       patterns:
         no-set-methods:
-          pattern: /def set_\w+/
+          pattern: "def set_\\w+"
```

Comment:
I'm afraid, it's not write pattern exactly as one would in terminal. YAML reserves `\` as an escape sequence starter. So we have to escape backslashes in the config. I can clarify that in the README but I can not change this without implementing an alternative YAML parer.